### DISCUSSION TITLE
v0.8.0 - Batch Automation

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#32] [General]
## What's Changed
* Image Batch Processing by @saddam213 in https://github.com/saddam213/OnnxStack/pull/23
* Add integration tests by @james-s-tayler in https://github.com/saddam213/OnnxStack/pull/25
* Remove Legacy Image Batching by @saddam213 in https://github.com/saddam213/OnnxStack/pull/26
* Added Async suffix to any methods that returned a Task or Task<> and … by @riddlemd in https://github.com/saddam213/OnnxStack/pull/28
* Changed LoadModel to LoadModelAsync in UnitTests by @riddlemd in https://github.com/saddam213/OnnxStack/pull/29
* Initial Float16 and BFloat16 onnx type  support by @saddam213 in https://github.com/saddam213/OnnxStack/pull/31
* Vectorize most of TensorHelper by @jdluzen in https://github.com/saddam213/OnnxStack/pull/15

## Nuget Packages
https://www.nuget.org/packages/OnnxStack.Core/0.8.0
https://www.nuget.org/packages/OnnxStack.StableDiffusion/0.8.0

## New Contributors
* @riddlemd made their first contribution in https://github.com/saddam213/OnnxStack/pull/28
* @jdluzen made their first contribution in https://github.com/saddam213/OnnxStack/pull/15

**Full Changelog**: https://github.com/saddam213/OnnxStack/compare/v0.7.1...v0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/saddam213/OnnxStack/releases/tag/v0.8.0'>v0.8.0 - Batch Automation</a>.</em>

**Comment by Amin456789** [comment]
nice thank u!
